### PR TITLE
Logrus error field must be set for Sentry to use the stack trace corr…

### PR DESCRIPTION
**Pull request by maroux**
…ectly

### CODE REVIEW

File: validator.go [hunk 1]
```diff
@@ -9,12 +9,12 @@ package hedwig
 
 import (
 	"encoding/json"
-	"errors"
 	"fmt"
 	"io/ioutil"
 	"regexp"
 	"strings"
 
+	"github.com/pkg/errors"
```
Comment (mngo87) [inline validator.go]: should also update the references to `fmt.Errorf` to use this new package

Comment (maroux) [reply validator.go]: Good call. Updated.

File: aws_client_test.go [hunk 2]
```diff
@@ -999,12 +999,12 @@ func (suite *AWSClientTestSuite) TestAWSClient_messageHandlerHookError() {
 	msgJSON, err := message.JSONString()
 	suite.Require().NoError(err)
 
-	expectedError := fmt.Errorf("Fake error!")
+	expectedError := errors.Errorf("Fake error!")
```
Comment (mngo87) [inline aws_client_test.go]: missing import for `pkg/errors`

Comment (maroux) [reply aws_client_test.go]: It's imported - tests are passing :/

Comment (mngo87) [reply aws_client_test.go]: oh guess diff just doesn't show all

**Pull request merged**